quickwit/quickwit-control-plane/src/control_plane.rs

```diff
@@ -76,10 +286,14 @@ impl Handler<GetOpenShardsRequest> for ControlPlane {
 
     async fn handle(
         &mut self,
-        _request: GetOpenShardsRequest,
+        request: GetOpenShardsRequest,
         _: &ActorContext<Self>,
     ) -> Result<Self::Reply, ActorExitStatus> {
-        unimplemented!()
+        match self.ingest_controller_mailbox.ask_for_res(request).await {
```

Comment:
I see some of the interaction with the ingest_controller is sync and some are not (we just post. a message, for instance on CreateIndex). 

Is there some benefit to the async ones?
If it does not matter much, I think the code would benefit from just making the Ingest controller a member of the control plane.

Reply:
ah actually I just saw your comment a little bit above.

Reply:
Good point. Processing events in a sync fashion will be easier to reason about. I will change `send_message` calls to `ask_for_res`.